src/services/formatting/rules.ts

```diff
@@ -213,25 +213,15 @@ namespace ts.formatting {
         public NoSpaceBetweenYieldKeywordAndStar: Rule;
         public SpaceBetweenYieldOrYieldStarAndOperand: Rule;
 
-        // Async-await
+        // Async functions
         public SpaceBetweenAsyncAndFunctionKeyword: Rule;
-        public NoSpaceBetweenAsyncAndFunctionKeyword: Rule;
-        public SpaceAfterAwaitKeyword: Rule;
-        public NoSpaceAfterAwaitKeyword: Rule;

```

Comment:
I once thought `NoSpace` rules would first remove those whitespaces and after that `Space` rules would add one. However, a `Space` rule can remove excessive whitespaces by itself.